SimpleServer/test.cc

```diff
@@ -40,27 +41,37 @@ using bricks::strings::Printf;
 
 TEST(LogCollector, Smoke) {
   std::ostringstream os;
-  LogCollectorHTTPServer collector(FLAGS_log_collector_test_port, os);
+  LogCollectorHTTPServer collector(FLAGS_log_collector_test_port, os, "/log", "OK\n", 100);
 
-  bricks::time::SetNow(static_cast<bricks::time::EPOCH_MILLISECONDS>(1234));
+  bricks::time::SetNow(static_cast<bricks::time::EPOCH_MILLISECONDS>(12));
   const auto get_response = HTTP(GET(Printf("http://localhost:%d/log", FLAGS_log_collector_test_port)));
   EXPECT_EQ(200, static_cast<int>(get_response.code));
   EXPECT_EQ("OK\n", get_response.body);
 
-  bricks::time::SetNow(static_cast<bricks::time::EPOCH_MILLISECONDS>(5678));
+  bricks::time::SetNow(static_cast<bricks::time::EPOCH_MILLISECONDS>(112));
+  std::this_thread::sleep_for(std::chrono::milliseconds(150));
```

Comment:
I'm fine keeping this, but it's scary. Please add a comment at least.

(I'd add an `atomic_size_t` for the total number of events published into `ostream_`, and wait on its value. `#ifdef`-guarded to be test-only, if/when performance is a concern.